### DISCUSSION TITLE
[2024-05-13] Team GUI Demo Day

### DISCUSSION THREAD

**Discussion by farmaazon** [enso-org/enso#9920] [Weekly Demo Day]
* #9874 - It all started from GeoMap being stuck on drag. When fixing it, we decided to refactor our node/scene drag handlers, so the panels don't need to stop pointer events. Also, it's possible to move node by grabbing some widgets e.g. checkbox or drop down (but not text/numeric)  - we discern if the user interaction is click (which is handled by widget) or drag (which moves the node).

[Screencast from 2024-05-08 10-45-33.webm](https://github.com/enso-org/enso/assets/3919101/d521b7a4-96c5-4e2c-a8a6-84388a18d041)

* #9827

![image](https://github.com/enso-org/enso/assets/3919101/29c09e74-47db-4bd5-8770-4a79b23a7cf1)

* #9837 - they are configurable, and now it's possible to resize left-side of visualization. They are narrower than in the design, because they covered scroll bars otherwise.

[Screencast from 2024-05-02 11-51-17.webm](https://github.com/enso-org/enso/assets/3919101/d58a0970-6994-4ac6-a55e-d77bccc6dac5)

* #9700 - it's now not clipped, but also there was a bug where it wasn't hidden by clicking at background (you needed to click inside visualization).

[Screencast from 2024-04-30 15-09-17.webm](https://github.com/enso-org/enso/assets/3919101/a11ae875-7e2f-4c84-8d91-33aa1ca8bdbd)

**Comment by kazcw** [comment]
- #9910

https://github.com/enso-org/enso/assets/1047859/6feb5d23-1525-48f7-933e-c9371312decf

- Use #9910 in component comments

![image](https://github.com/enso-org/enso/assets/1047859/c5df13fe-0290-4f1d-abb2-b2f42df274d3)

- #9825

https://github.com/enso-org/enso/assets/1047859/c3eff168-6807-4825-b17b-053e3cd8b04c

- #9842
- #9913

**Comment by vitvakatu** [comment]
Quite a short iteration for me:

- https://github.com/enso-org/enso/pull/9916 - a small fix for autoscoped constructors.
- https://github.com/enso-org/enso/pull/9909 - a huge change to the placement of the dropdown arrows.


https://github.com/enso-org/enso/assets/6566674/e3ba96b3-fff8-4553-ac94-866436aea013

**Comment by JaroslavTulach** [comment]
Related engine work:
- #9528
- #9862
- #9808
- Startup time may go dow to 33% easily: #9866
- #9797
- #9916

**Comment by mwu-tow** [comment]
This week summary:
* New, custom installer and uninstaller were completed and merged. They replace old ones based on NSIS (through electron-builder). #9815
* This enabled finally bumping electron-builder to the latest version, dropping the `python2` requirement on macOS and simplifying the CI workflow definitions. #9884
* Fix CI builds for third-party contributors (fork-based PRs). #9926
* Updating the self-hosted runners (all Linux and Windows machines, skipped the macOS one). 

Minor cleanups:
* Removing unused or commented-out code and unnecessary dependencies. #9885
* Removed duplicated workflow definition. #9888

**Comment by Frizi** [comment]
This iteration my time was highly reduced due to sick leave and recovery.

- Currently working on new event logger that sends telemetry to the opensearch backend through cloud API #9730
